mdx/emit.ts

```diff
@@ -86,7 +86,66 @@ const CATEGORY_LABELS = {
     promises.push(
       emitMessagesJson({
         filePath: pathToPlugin,
-        messages: pkg.messagesData,
+        messages: pkg.messagesData.concat(
+          // Concat with this array because we'll need the inner messages
+          // in the dictionary for syntax coloring.
+          //
+          // Each array element is in this form, for example:
+          //
+          // {
+          //   name: "Location";
+          //   packageName: "location.v1";
+          //   hash: "location";
+          //   body: `
+          //    <Definition>
+          //
+          //    <DefinitionHeader name="message">
+          //
+          //    ### Booking
+          //
+          //    </DefinitionHeader>
+          //
+          //    Represents the booking of a vehicle.
+          //
+          //    Vehicles are some cool shit. But drive carefully.
+          //
+          //    ```protosaurus--booking.v1.Booking
+          //    message Booking {
+          //      // ID of booked vehicle.
+          //      int32 vehicle_id = 1;
+          //      // Customer that booked the vehicle.
+          //      int32 customer_id = 2;
+          //      // Status of the booking.
+          //      BookingStatus status = 3;
+          //      // Has booking confirmation been sent.
+          //      bool confirmation_sent = 4;
+          //      // Has payment been received.
+          //      bool payment_received = 5;
+          //      // Color preference of the customer.
+          //      string color_preference = 6;
+          //      // Pick-up location.
+          //      // This is a coordinate.
+          //      Location pickup_location = 7;
+          //      // Destination location.
+          //      // This is a coordinate.
+          //      Location destination_location = 8;
+          //      // Intermediate locations.
+          //      repeated Location intermediate_locations = 9;
+          //    }
+          //    ```
+          //
+          //    </Definition>
+          //   `;
+          // }
```

Comment:
@dio I have added the sample content of the `MessageData`, WDYT?

Reply:
SG